march_gain_scheduling/src/march_gain_scheduling/dynamic_pid_reconfigurer.py

```diff
@@ -65,6 +64,12 @@ def client_update(self):
                                                                 self.current_gains[i][1],
                                                                 self.current_gains[i][2]))
 
+    def load_current_gains(self):
+        self.current_gains = []
+        for joint_index in range(len(self._joint_list)):
```

Comment:
why not just use a normal for loop (also applicable for line 17!):
```suggestion
        for joint_name in self._joint_list:
```